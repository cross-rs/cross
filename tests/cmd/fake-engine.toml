# NOTE: `stderr` has a lot of inline replacements for bash escape codes
bin.name = "cross"
args = ["build", "--color", "never"]
status = "failed"

[env.add]
CROSS_CONTAINER_ENGINE = "cross-fake-engine"
