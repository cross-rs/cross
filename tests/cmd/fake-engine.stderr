Error: 
[..]no container engine found[..]

Location:
[..]src/docker/engine.rs[..]:[..]

[..]Suggestion[..]: is docker or podman installed?[..]

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
